### PR TITLE
[Teams] Browser Isolation + Tenant Control

### PR DESCRIPTION
**Pull request by abracchi-tw**

### CODE REVIEW

File: products/cloudflare-one/src/content/policies/filtering/http-policies/tenant-control.md [hunk 1]
```diff
@@ -87,4 +87,43 @@ This section covers policy configurations for common SaaS applications.
...
 
-</TableWrap>
\ No newline at end of file
+</TableWrap>
+
+## Using Tenant Control with Browser Isolation
+
+Browser Isolation may be configured to send custom request headers. This is useful for implementing SaaS application Tenant Control or sending arbitrary custom request headers to Isolated websites.
+
+This is achieved by implementing two HTTP Policies targettin the same domain or application group in the Cloudflare for Teams dashboard.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/policies/filtering/http-policies/tenant-control.md]:
```suggestion
This is achieved by implementing two HTTP Policies targeting the same domain or application group in the Cloudflare for Teams dashboard.
```

File: products/cloudflare-one/src/content/policies/filtering/http-policies/tenant-control.md [hunk 2]
```diff
@@ -87,4 +87,43 @@ This section covers policy configurations for common SaaS applications.
...
+
+### Example: Implementing a custom request header for a domain
+
+#### 1. Create an Isolate policy
+
+* **Name**: Isolate HTTPBIN
+* **Description**: Isolates the h`ttpbin.org` website.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/policies/filtering/http-policies/tenant-control.md]:
```suggestion
* **Description**: Isolates the `httpbin.org` website.
```

File: products/cloudflare-one/src/content/policies/filtering/http-policies/tenant-control.md [hunk 3]
```diff
@@ -87,4 +87,43 @@ This section covers policy configurations for common SaaS applications.
...
+| -- | -- | -- |
+| Domain | in | httpbin.org |
+
+  ![Isolate policy](../../../static/documentation/policies/httpbin-policy-1.png)
+
+
+#### 2. Create an Allow policy with a Custom Header
+
+* **Name**: Inject custom request header for HTTPBIN
+* **Description**: Adds a custom request header for all traffic to the httpbin.org website.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/policies/filtering/http-policies/tenant-control.md]:
```suggestion
* **Description**: Adds a custom request header for all traffic to the `httpbin.org` website.
```

**Review by marciocloudflare — APPROVED** [review]

**Comment by abracchi-tw** [comment]
@obezuk we'll merge as soon as you approve. thanks! :)

**Pull request merged**